warre/inventory/additem.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
```

Comment:
This file is throwing a bunch of html validation warnings, can you take a look at that?

https://validator.w3.org/nu/#file